DICUI.Test/Utilities/DriveTest.cs

```diff
@@ -0,0 +1,21 @@
+﻿using DICUI.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+
+namespace DICUI.Test.Utilities
+{
+    public class DriveTest
+    {
+        [Fact]
+        public void DriveConstructorsTest()
+        {
+            Assert.True(Drive.Floppy('a').IsFloppy);
+            Assert.False(Drive.Optical('d', "test").IsFloppy);
```

Comment:
Only issue here is that on each unique computer that these tests run on, these tests may or may not pass. For example, on mine, I don't have A connected at all times, so it would fail when this test runs.

Reply:
This test is not for an effective drive but just for the construction of the `Drive` object, nothing from the machine is actually used.

Reply:
Oh okay, just making sure.